upstream-community-operators/storageos/storageos.v0.0.10.clusterserviceversion.yaml

```diff
@@ -0,0 +1,267 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: storageosoperator.v0.0.10
+  namespace: placeholder
+  annotations:
```

Comment:
The `metadata.annotations.containterImage` field needs to be added with the operator's container image address. This should match the image used in the deployment section, ie: `storageos/cluster-operator:latest`

Reply:
Thanks. I've added that.
That reminds me that the containers in a CSV release should be a specific version and not latest, for reproducibility. I've set the container image tag to v0.0.11.